core/src/main/java/org/elasticsearch/tasks/TaskResult.java

```diff
@@ -160,7 +160,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
     public XContentBuilder innerToXContent(XContentBuilder builder, Params params) throws IOException {
         builder.field("completed", completed);
-        builder.field("task", task);
```

Comment:
of course the problem here is anyways that TaskInfo is not a ToXContentObject ...